mev-boost-relay/mev-boost-relay.go

```diff
@@ -156,6 +156,11 @@ func New(config *Config) (*MevBoostRelay, error) {
 		return nil, fmt.Errorf("incorrect builder API secret key provided")
 	}
 
+	// rbuilder now requires the relay to support bundle cancellations
+	if err := os.Setenv("ENABLE_BUILDER_CANCELLATIONS", "1"); err != nil {
```

Comment:
This is already enabled here https://github.com/flashbots/builder-playground/pull/38/files#diff-ba64767ec1e036f2cf2c2b2f72b58f59bd94244b70c5cce38b0b7dbe91d4eedcR67

Reply:
Ah I hadn't pulled recently so wasn't aware of https://github.com/flashbots/builder-playground/commit/56db5d5585151b51caf90ff5df792a179d6397c4 👍